### DISCUSSION TITLE
Limit battery charging to prolong battery life

### DISCUSSION THREAD

**Discussion by VirusBurst** [hanai3Bi/Switch-OC-Suite#55] [General]
TBH for me an important topic since my switch v2 is almost 99% of it's time docked therefore I was happy, that I could limit the charging from 20% to 80% with just 500mAh...

I see, that sys-clk removed this feature for reasons and I'm asking myself, whether this feature will return sooner or later.

There was a guide about writing something into the config file of atmosphere but this didn't have any effect :(

**Comment by hanai3Bi** [comment]
arbitrary control of charging limit/current in old sys-clk-oc caused various prolems, so they were removed altogether while transition to new sys-clk.

HOS by default sets 500mAh for charging when in docked (old sys-clk-oc was overriding this with 2A).
and honestly i would just not worry about battery life and replace with new one if it is dead. i always play with charger connected and battery life is still fine.
